src/language-html/is-source-element.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+function isSourceElement(node, parentNode, opts) {
+  if (opts.parser === "vue") {
```

Comment:
Instead of checking `parser`, let's only add this to `vue` parser?  https://github.com/prettier/prettier/pull/9680/files#diff-a1f317647562c82c569f0bab8378f06eebc7ea49703be03385fa440e656c36a5R403